install.sh

```diff
@@ -10,9 +10,10 @@ WHITE="\\033[37;1m"
 
 print_unsupported_platform()
 {
-    >&2 say_red "error: We're sorry, but it looks like the Dashwave CLI is not supported on your platform"
-    >&2 say_red "       We support 64-bit versions of Linux and macOS and are interested in supporting"
-    >&2 say_red "       more platforms.  Please reach out to us at hello@dashwave.io"
+    >&2 say_red "error: We're sorry, but it looks like installation of the Dashwave CLI"
+    >&2 say_red "       using this script is not supported on your platform. We support"
+    >&2 say_red "       debian based Linux and are interested in supporting more"
+    >&2 say_red "       platforms. Please reach out to us at hello@dashwave.io"
```

Comment:
Add a line
`If you are using MacOS, prefer installing using the brew package manager. Refer to the docs.`

Reply:
thought behind not adding that line :: we already have installation stepps for macos users in docs, and for other os or distro users this msg would be a kill

Reply:
I don't think a non debian based linux user getting this msg makes any sense, basically no specific messaging when things are generic.

Reply:
I get it, but the intention of this message is to guide the users that there is a preferred way to install on MacOS, which you can find in the docs, because this error most probably from MacOS users right now.
If you want, you can do a `platform` check in the script if you don't want to show this message to who aren't either Debian or Darwin users